src/renderers/dom/shared/ReactDOMInstrumentation.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMInstrumentation
+ */
+
+'use strict';
+
+var ReactDOMDebugTool = require('ReactDOMDebugTool');
+
+module.exports = {debugTool: ReactDOMDebugTool};
```

Comment:
Yes, eventually.  Right now it's controlled by `__DEV__`, and is not as simple as just replacing all `__DEV__`s with a feature flag for reasons around dead code elimination.